sdcm/sct_config.py

```diff
@@ -1444,9 +1431,9 @@ def _check_backend_defaults(backend, required_params):
                 raise ValueError(error_message_template.format('<max PK value> should be bigger then <min PK value>. '))
 
         # verify that the AMIs used all have 'user_data_format_version' tag
-        ami_id_db_scylla = self.get('ami_id_db_scylla', default='').split()
-        region_names = self.get('region_name', default='').split()
-        ami_id_db_oracle = self.get('ami_id_db_oracle', default='').split()
+        ami_id_db_scylla = (self.get('ami_id_db_scylla') or '').split()
```

Comment:
2) put the `ami_id_db_scylla: ''` and `ami_id_db_oracle: ''` to aws_defaults.yaml
3) remove or ' '
4) no need for default for `region_name` it is checked before

Reply:
done